src/cf-buttons/src/atoms/buttons.less

```diff
@@ -72,13 +74,14 @@
   }
 
   &:hover,
-  &.hover {
+  &.hover,
+  &:focus,
```

Comment:
Huh. Thanks for digging in and seeing how it was before. I guess it's trying to be extra cautious about the cascade. :+1: for even the minorest of gains, we do need to get our selector count down. 🇺🇸